src/harness/util.py

```diff
@@ -277,3 +279,14 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+
+
+def countdown(t):
```

Comment:
I don't think this is used anywhere in this PR.

Reply:
This   file has been removed as the associated test case has been removed.